.github/workflows/unit_tests.yml

```
@@ -14,8 +14,6 @@ jobs:
       python_version: "3.8"
   docker_build_tests:
     uses: neongeckocom/.github/.github/workflows/docker_build_tests.yml@master
-    with:
-      runner: self-hosted
```

These tests fail on some GitHub-hosted runners (noted in #148)
```suggestion
    with:
      runner: self-hosted
      platforms: linux/amd64,linux/arm64
```